### ISSUE TITLE
Make the markdown renderer include non-ascii characters in channel names

### ISSUE THREAD

**Issue by Powersource** [ssbc/patchwork#998]
The title is maybe a bit unclear but my specific problem is that when you make a message saying "#malmö" it only renders "#malm" as clickable (and it takes you to the malm channel).

Everything else around this seems to work fine, I can go to the channel using the search box, subscribe to it, and the post linking "#malm"(ö) even ends up in the "#malmö" channel.

I have a feeling this is going to have to be fixed in a dep but I haven't had time to look into that yet.

**Comment by christianbundy** [comment]
@Powersource 

Yep, this is in ssb-markdown, I'll transfer this issue now. I'd love to conform to Unicode [hashtag identifier syntax](http://unicode.org/reports/tr31/#hashtag_identifiers). I think we can change the hashtag regex [here](https://github.com/ssbc/ssb-markdown/blob/fce5cf028a3fb884590ff525040f52f25a95087c/lib/block.js#L30).
